twyla/service/event_bus.py

```diff
@@ -1,6 +1,15 @@
 from aioamqp.protocol import OPEN
 
 from twyla.service import queues
+from twyla.service.event import Event
+
+class MessageToEventAdapter:
+    def __init__(self, callback):
+        self.callback = callback
+
+    async def __call__(self, channel, body, envelope, properties):
```

Comment:
`properties` isn't used. What is it for?

Reply:
The signature is required by aioamqp for callback functions. Instead of using `_`, the default name for unused arguments, I prefer leaving it in with the real name, in case it is used in the future.

Reply:
👍